### ISSUE TITLE
The page remains in a loading state when opening an existing tab or switching tabs when the libraries are opened

### ISSUE THREAD

**Issue by Softvision-RemusDranca** [MozillaReality/FirefoxReality#2889]
**[Notes]:**
- As a side effect of this issue, if the affected window is closed the background remains dimmed [video](https://drive.google.com/open?id=1ox4VgXTBGRlhjHl6gm0o0PYKydVDZxVA).

**[Affected Versions]:**
- Firefox Reality 9 RC 4 (c64365e3)

**[Affected Platforms]:**
- HTC Vive Focus Plus - Android 7.1.2
- HTC Vive Focus - Android 7.1.2
- Oculus Quest - Android 7.1.1
- Oculus Go - Android 7.1.2
- Pico Neo 2 - Android - 8.1.0

**[Prerequisites]:**
- The headset is turned on, the browser is installed and open.
- Have 2 windows opened, and multiple tabs.

**[Steps to reproduce]:**                                        
1. Click the "History" button.
2. Click the "Tabs View" button.
3. Select a different tab than the ones opened.
4. Observe the behavior.

**[Expected results]:**
- The tab is correctly displayed.

**[Actual results]:**
- The window remains in a loading state. 

**[Additional Notes]:**
- The issue is also reproducible while the Bookmarks library is opened.
- The behavior on Oculus Quest is slightly different but more damaging for which I will log a separate issue for tracking purpose. 
- The issue is reproducible even when tabs are switched between windows, while a library is opened.
- The wrong URL is displayed in the affected tab.
- The issue is dismissed if the window is refreshed.
- This issue is not reproducible if a new tab is opened.
- Attached are screen recordings of the issue:
	- Open existing tab: [link](https://drive.google.com/open?id=1wLc3IovqZBn19qjiXbLZtn3-SYvj-g2-).
	- Switch tabs active window: [link](https://drive.google.com/open?id=1LkrvdCVmnqt-Afj1q4jMIhGWekw77QUD).

**Comment by Softvision-RemusDranca** [comment]
I have verified this issue with the v9 RC5 (3e30ba6) build and it's partially fixed. The issue where opening an existing tab is no longer reproducible.
However, the issue where switching tabs while the libraries are opened it's still reproducible:
- On Pico Neo 2 and HTC devices is always reproducible.
- On Oculus Go is intermittent.
- On Oculus Quest only the side effect where the wrong URL is displayed in the affected tab is reproducible.

Considering this I will reopen this issue.

**Comment by Softvision-RemusDranca** [comment]
I have verified this issue and it's no longer reproducible with the Firefox Reality 9 rc6 (abe931f) build. Now the libraries are dismissed and the websites are correctly displayed in both cases (opening an existing tab or switching tabs).

Tested on Oculus Go (Android 7.1.2), Oculus Quest (Android 7.1.1), HTC Vive Focus Plus (Android 7.1.2), HTC Vive Focus (Android 7.1.2) and Pico Neo 2 (Android 8.1.0).
